go.mod

```diff
@@ -1,9 +1,9 @@
 module chainguard.dev/apko
 
-go 1.23.2
+go 1.23.4
```

Comment:
Is this required because of something, or just bumping?

Reply:
it needed `1.23.3`, i figured might as well bump it to latest, as thats what i have installed